cl/people_db/import_judges/judge_utils.py

```diff
@@ -142,6 +142,7 @@ def get_degree_level(degstr):
 
 def get_party(partystr):
     partydict =  dict([(v,k) for (k,v) in [('d', 'Democrat'),
+        ('d', 'Democratic'),
```

Comment:
Huh, how was this missing before? Any ideas?

Reply:
Don't know.  Maybe the FJC spreadsheets were "cleaned" before they were ingested?